recipes/seismon/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "seismon" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/seismon-{{ version }}.tar.gz
+  sha256: ceff57b53e3463a84408425f0ad3feba2c5eb181bef3176af41e5aca15274a23
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
```

This project needs `setuptools` to build

```suggestion
    - pip
    - python
    - setuptools
```